packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/Dashboard.page.tsx

```diff
@@ -46,7 +46,7 @@ const Dashboard = () => {
 
   useEffect(() => {
     const filteredVolume: ai.volume.Volume[] = notebook.spec.volumes.filter(
-      (vol) => vol.volumeSource.dataStore.internal === false,
+      (vol) => vol.mountPath !== '/workspace',
```

Comment:
Can you reuse your config from volume.const.ts here ?

Reply:
Yep Done